### PR TITLE
Make phase banner configurable

### PR DESCRIPTION
**Pull request by JonF1**
Allows the tag and text attributes of the phase banner component to be configurable

### CODE REVIEW

File: spec/components/govuk_component/configuration/phase_banner_component_configuration_spec.rb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+      let(:overridden_default_text) { 'Public service' }
+
+      before do
+        Govuk::Components.configure do |config|
+          config.default_phase_banner_component_tag = overridden_default_tag
+          config.default_phase_banner_component_text = overridden_default_text
+        end
+      end
+
+      subject! { render_inline(GovukComponent::PhaseBannerComponent.new(**kwargs)) }
```
Comment (peteryates) [inline spec/components/govuk_component/configuration/phase_banner_component_configuration_spec.rb:22]: I think it makes sense to drop `kwargs` here, and above (lines 4-6), they're not actually used for anything.

**Review by peteryates — APPROVED** [review]
Looks great to me. Left one comment on removing a few variables we perhaps don't need.

**Comment by peteryates** [comment]
Nice, good to go! Thanks @JonF1

**Comment by JonF1** [comment]
> Nice, good to go! Thanks @JonF1

No problem 👍

**Pull request merged**